pkg/services/govmomi/vcenter/clone.go

```diff
@@ -369,6 +371,18 @@ func getDiskSpec(vmCtx *capvcontext.VMContext, devices object.VirtualDeviceList)
 			diskSpecs = append(diskSpecs, additionalDiskConfigSpec)
 		}
 	}
+
+	// Now if we have increased the disk size of any additional disks that were in the template, we can now add new disks
+	// that are present in the additionalDisks list.
```

Comment:
```suggestion
	// If there are DataDisk in the VM template create disk specs for them too.
```
seem clearer

but i think this sentence is confusing. please revise it.

Reply:
Sure.  The intent was for me to remember that we got to handle the OVA template disks first before doing the additional disks.

Reply:
I updated the comment to say 
`Process all DataDisks definitions to dynamically create and add disks to the VM`